packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal-docs.json

```diff
@@ -1534,6 +1497,43 @@
         }
       }
     },
+    "/structural-variant-genes/fetch": {
+      "post": {
+        "tags": [
+          "Study View"
+        ],
+        "summary": "Fetch fusion genes by study view filter",
```

Comment:
Could you update the backend description here?

Reply:
Sure